NAMESPACE

```diff
@@ -16,6 +16,7 @@ export(convert_01_ny)
 export(convert_bib_to_list)
 export(convert_list_to_bib)
 export(convert_month_range_string_to_binary)
+export(convert_month_range_string_to_binary_worker)
```

Comment:
Generally I wouldn't export a function like this, designed to run inside another function. Do we need to export it? If not remove export

Reply:
Yeah I agree, probably doesnt need to be exported. It was throwing an error in build because it wasnt exported which is why I added it. Probably can use austraits:::convert_month_range_string_to_binary_worker  in the tests to get around not exporting it.

Reply:
Yes, good idea!